rules/prefer-math-min-max.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+const {fixSpaceAroundKeyword} = require('./fix/index.js');
+
+const MESSAGE_ID = 'prefer-math-min-max';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` to simplify ternary expressions.',
+};
+
+/**
+@param {import('eslint').Rule.RuleContext} context
+@param {import('estree').ConditionalExpression} node
+@param {import('estree').Node} left
+@param {import('estree').Node} right
+@param {string} method
+*/
+function getProblem(context, node, left, right, method) {
+	const {sourceCode} = context;
+
+	// Catch edge case: `(0,foo) > 10 ? 10 : (0,foo)`
+	const getText = n => n.type === 'SequenceExpression' ? `(${sourceCode.getText(n)})` : sourceCode.getText(n);
+
+	return {
+		node,
+		messageId: MESSAGE_ID,
+		data: {replacement: `${method}()`},
+		/** @param {import('eslint').Rule.RuleFixer} fixer */
+		* fix(fixer) {
+			/** @type {{parent: import('estree'.Node)}} */
+			const {parent} = node;
+			if (
+				// Catch edge case: `return+foo > 10 ? 10 : +foo`
+				(parent.type === 'ReturnStatement' && parent.argument === node && parent.start + 'return'.length === node.start)
```

Comment:
We should remove the AST check here, `fixSpaceAroundKeyword` handles more cases.